internal/cmds/cmds.go

```diff
@@ -10,6 +10,7 @@ const (
 	mtGetTag = uint16(1<<11) | readonly // make mtGetTag can also be retried
 	scrRoTag = uint16(1<<10) | readonly // make scrRoTag can also be retried
 	unsubTag = uint16(1<<9) | noRetTag
+	pipeTag  = uint16(1 << 8) // make blocking mode request can use auto pipelining
```

Comment:
```suggestion
	pipeTag  = uint16(1<<8) // make blocking mode request can use auto pipelining
```
Just aligning the style above.